lib/pong.js

```diff
@@ -321,6 +321,8 @@ var pong = {
                 losers = challenge.challenger;
                 winners = challenge.challenged;
               }
+              challenge.winner = winners;
+              challenge.save();
```

Comment:
This is async and may fail, right? If you're going to save something you should save both losers and winners IMO.This also needs tests.